include/roboteam_ai/world_new/views/WorldDataView.hpp

```diff
@@ -188,7 +188,7 @@ class WorldDataView {
      * @param team Team enum of team to fetch from
      * @return A non-owning view of the robot that has the ball
      */
-    [[nodiscard]] RobotView whichRobotHasBall(Team team = both);
+    [[nodiscard]] std::optional<RobotView> whichRobotHasBall(Team team = both, double maxDist = ai::Constants::MAX_BALL_BOUNCE_RANGE());
```

Comment:
Can you update the params doc for this function? It is not clear to me what the max_ball_bounce_range has to do with which robot has the ball

Reply:
I don't know this either. This was already used in the "old" world. I just copied it

Reply:
I will adjust it

Reply:
I think this should not be MAX_BALL_BOUNCE_RANGE, but rather MAX_BALL_RANGE, no?

Reply:
I'm not sure, that might be the case. It kind of depends on why you want to call the function I guess, which is why you can give your own minimum distance.

Reply:
I pushed a commit where I changed the default to MAX_BALL_RANGE so I guess its fine